common/transform/build.gradle

```diff
@@ -14,6 +14,7 @@ dependencies {
     testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api'
     testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-params'
     testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine'
+    testImplementation group: 'uk.org.webcompere', name: 'system-stubs-jupiter', version: '1.2.0'
```

Comment:
Consider dropping the version here, and instead adding version to https://github.com/h2oai/dai-deployment-templates/blob/master/gradle.properties and https://github.com/h2oai/dai-deployment-templates/blob/master/gradle/mixins/dependencies.gradle.

That way, dependency versions are centralized and shared across projects.